buildSrc/build.gradle

```diff
@@ -10,17 +10,32 @@ boolean localRepo = project.getProperties().containsKey("localRepo")
 
 Properties props = new Properties()
 props.load(project.file('esh-version.properties').newDataInputStream())
-version = props.getProperty('eshadoop')
+String eshVersion = props.getProperty('eshadoop')
 String esVersion = props.getProperty('elasticsearch')
 
+// TODO: This default should be removed when ES stops using alpha1 as its default qualifier
+String qualifier = System.getProperty("build.version_qualifier", "alpha1")
```

Comment:
Does the release manager need to be aware of this system property ?

Reply:
There's a separate PR open for updating the release automation with the new qualifier code.